### ISSUE TITLE
Julio Perez

### ISSUE THREAD

**Issue by aflies1** [promazo/Sourcing#186]
Mex. Dev

**Comment by aflies1** [comment]
Why are you passionate about coding?

**Comment by aflies1** [comment]
Why are you passionate about coding?

**Comment by aflies1** [comment]
English - 10/10
Current job? Working on a large project or doing the same thing?
- has been moving up to different projects. 
- current companies brings in different projects, created some iterations to QuickBooks (React) but a lot of things involved with that work
- His other work has been in backend - tax area and payment fields

**Comment by ldaudet** [comment]
Julio Armando Perez Gama - 9/19/22 - 9am

His english is really really good.

Full stack developer, considered himself a bit skewed toward backend. Working with React a little bit more than 4 years. Started small application for small companies until he joined Invalsa. Then he moved to Iterando, which was a consultancy company, making apps for other companies. He then moved to Kavak, the first mexican unicorn, a clone of carvana. He worked there as a backend dev. He used graphQL, Lambda, typescipt, and jest for the testing.

Finally, he worked for ParallelStaff, where he worked with React, quickbooks, testing with cypress. That's where he got his english as they were US company.

He started coding when he was 11, making some html pages. Then he moved in college, where he exceled at discreet math. He was exposed to programming then and saw he had potential. He passed the exams and never turned back. 

As far as backend, he got a good exposure to AWS. He hasn't worked with Amplify in the past. 

He worked on a tool for Alexa that tell his girlfriend good morning I love via an ngrock tunnel from his personal server. When he was in school he created an iphone app using java. He also created a watch application for a contest. He didn't win the prize but he really liked it.

He worked with React and Flutter in the past but not for react native. He never worked as a mobile dev native dev either. Flutter is the only experience he's got in mobile dev.

Never used DynamoDB in the past, but he knows graphql for having used it.

He would like more to be specialized in React, though not closed to the idea of learning react native.

He is really good at git and an advanced user.

TL;DR: could be a great addition to the team

**Comment by mikepromazo** [comment]
Summary: His english was conversational but he definitely struggled during the interview to find certain words or got caught up in phrasing.  This could have been because he was nervous. This may have led to less detailed answers.  I didn't get much more than a few sentences for each of my questions.

He seems green, especially when compared to Louis and Arturo.  He graduated in 2021 so has less than 2 years of full time experience.  I didn't get the sense he has as much independent experience where he was working more on his own or driving an effort forward as he's always been with a team, and in some cases a larger team. After reading Louis' notes on his tech background it doesn't sound like there is an exact alignment on our tech stack.  However, I'm not sure if what he does have experience in translates super easily to our stack.

He seems genuinely interested in ProMazo but he didn't do much research outside what Louis and Adam told him.  He said he's interested in a startup and the idea of being in a hurry, trying to grow, things can change fast.  He doesn't have experience with one though and I didn't get the sense he fully understood the challenges that come with working at a startup.

Since he is younger, he could have a lot of upside if he's good technically and sticks with it.  Almost all of his past experience was ~1 yr or less because it was project based so I'm concerned he might jump if a better opportunity comes up.  We would need to better understand his alignment with PM.

We need Sr. Devs and Arturo is our new standard so we should hire Arturo's or better.  Julio does not seem to be at Arturo's level so I don't think we should hire him unless his tech chops are off the charts. 

- Young, graduated in 2021
- Resume has at least 2 typos (multi space, extra letters)
    - Initial notes
        - Camera was on the side of his face because of his laptop setup.  Looked somewhat disheveled
        - Seemed to struggle with some english, hesitated at several points
        - Coding is passion, searching how to solve problems
        - Started coding at 11, knew he wanted to do it long term
    - Background
        - Started talking about ProMazo, had learned from Adam/Louis.  Didn't reference other research he had done
        - Looks for companies he feels good about, gets excited
    - Types of projects he’s been on
        - Creasoft - data, evironmental, used internally
        - Invasa - 1st big project - created software for loan “things” related to buildings
        - Iterando - consultancy, some big projects for unis in mexico; apps for employees
        - Kavak - microservices, payment services
        - Boundless - 1st US based.  Integrations for other companies.  Said was a little confusing
    - Why did he leave each of the companies (max time ~10 months with any company but he did do ~1.5 years with Creasoft)?
        - Moved companies because the project was done
        - Looking forward to long term project
        - Why leaving the current company after less than a year?  Project ended
- **Knew proj was ending, didn’t line something up or get recruited.  Is this a red flag?**
    - Experience with different teams, team sizes, dynamics
        - Biggest = Kavak - 500 devs; 10 ppl per squad
        - Boundless - 4 devs, 6 mgr positions
        - Smallest = Imbasa - 4 devs, 1 proj mgr
        - Exp with smaller teams?
            - More comms, fewer comms problems.  Bigger teams have issues
            - Cons - if need a lot done having smaller team is tough
    - How does he manage being completely remote?
        - He’s worked both remote and onsite.  Onsite easier to be more engaged.  Said liked idea of having 4 hr working session
        - What does he like?
            - Likes it, has his setup he likes; more productive with his setup.  Very comfortable.  When onsite he lacks what he has
    - Most challenging work
        - Kavak had lots of squads, created something on own, then pulled together, was a mess.  Not tested properly; had to deploy hot fixes
        - Learned - have more comm with teams, channel to announce what they’re gonna do.  Mtg with all teams to see how it will be dpeloyed
    - What's driving his job search and decision making?
        - Important to work in place where he feels he’s creating someting useful.  Saw Louis is really good, could work together really well
        - Proj, technology, ability to grow; abililty to move into diff tech (react to react native)
    - Why PM?  
        - Really likes idea, said he would’ve used i, can help a lot of ppl in univ
        - Main reason he’s interested; wants to be part of idea he thinks will help ppl
    - His goals?
        - Wants to grow as dev, wants to be able to mentor them, lead them; tech lead
    - Pro/con of startup, his opinion
        - Likes ambiance of startup, everyone is in hurry, things can change fast.  Watching how it grows, being a part of it.  Seeing other developers join, bring them onboard
        - Concerns - maybe creating something we put a lot of effort into, maybe doesn’t happen
        - Didn’t seem like he had grasp/exp of being a part of a startup
    - His questions - asked several questions about early PM, stage of growth

**Comment by ldaudet** [comment]
Meeting with him 10/12/2022 for tech review
